src/app/medInria/areas/browser/medBrowserArea.cpp

```diff
@@ -91,15 +88,17 @@ medBrowserArea::medBrowserArea(QWidget *parent) : QWidget(parent), d(new medBrow
         d->sourceSelectorToolBox->setCurrentTab(1);
     }
 
-    //dataSources
-    foreach (medAbstractDataSource *dataSource, medDataSourceManager::instance()->dataSources())
+    // DataSources
+    for(medAbstractDataSource *dataSource : medDataSourceManager::instance()->dataSources())
```

Comment:
this was not supposed to be in 3.1. Did you push the differences between master and 3.1 on 3.1? It could be a problem